packages/aragon-cli/package.json

```diff
@@ -113,45 +111,62 @@
     "eslint-plugin-prettier": "^3.0.0",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
-    "husky": "^3.0.0",
-    "lint-staged": "^9.1.0",
     "nyc": "^14.1.1",
     "prettier": "^1.15.3",
     "proxyquire": "^2.1.0",
-    "regenerator-runtime": "^0.13.3",
     "sinon": "^7.3.1"
   },
+  "aragon": {
+    "clientRepo": "https://github.com/aragon/aragon",
+    "clientVersion": "775edd606333a111eb2693df53900039722a95dc",
+    "clientPort": "3000",
+    "defaultGasPrice": "2",
+    "defaultIpfsTimeout": "300000"
+  },
   "engines": {
     "node": ">=8.0.0"
   },
   "ava": {
     "require": [
-      "core-js",
-      "regenerator-runtime",
       "@babel/register"
     ],
     "files": [
       "test/**/*.test.js"
     ]
   },
-  "aragon": {
-    "clientRepo": "https://github.com/aragon/aragon",
-    "clientVersion": "775edd606333a111eb2693df53900039722a95dc",
-    "clientPort": "3000",
-    "defaultGasPrice": "2",
-    "defaultIpfsTimeout": "300000"
+  "babel": {
```

Comment:
Are we good with the configs we have for babel, eslint and prettier? Do we need to update something?

Reply:
I think they're fine so far, the only thing I added was:

```
          "targets": {
            "node": true
          }
```

which made the compile files a bit smaller and easier to read since we don't need to polyfill a bunch of stuff that browsers are missing.